test/src/gateways/postgres/plugin.test.js

```diff
@@ -1,10 +1,11 @@
 import pg from "pg";
-import PostgresGateway from "../../../../src/gateways/postgres/plugin";
+import PostgresGateway from "../../../../src/gateways/postgres/index";
 import sinon from "sinon";
 import { expect } from "chai";
 
 describe("Gateway Plugin Test", function () {
   const db = new PostgresGateway();
+  const error = new Error("Fake Error");
```

Comment:
**Annotation:** Placing the error constant to an outer scope reduces "DRY" code.